cfgov/jinja2/v1/_includes/molecules/related-topics.html

```diff
@@ -0,0 +1,45 @@
+{# ==========================================================================
+
+    Related Topics
+
+    ==========================================================================
+
+    Description:
+
+    Creates related topics markup when given:
+
+    value.heading: The heading of the related topics.
+                   (Default: Related Topics)
+
+    value.topics:  An list of links with the url first followed
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/127e98d41fd908bc46a3b3860431e3edc2edd179